DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/CourseDelegates/DelegateProgress.cshtml

```diff
@@ -0,0 +1,64 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.CourseDelegates
+@using Microsoft.Extensions.Configuration
+@model DelegateProgressViewModel
+
+@{
+  ViewData["Title"] = "Delegate progress";
+  ViewData["Application"] = ApplicationType.TrackingSystem.Name;
+  ViewData["HeaderPath"] = ApplicationType.TrackingSystem.HeaderPath;
+  ViewData["HeaderPathName"] = ApplicationType.TrackingSystem.HeaderPathName;
+  var routeParamsForBackLink = new Dictionary<string, string?> {
+    {
+      "customisationId", Model.CustomisationId.ToString()
+    }
+  };
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />

```

Comment:
Let's use our new `_AutoNavMenuItems` instead.
If you extract `ApplicationType.TrackingSystem` as a variable, you can pass it in, and you'll get a TrackingSystem navbar.